src/docfx/template/TemplateEngine.cs

```diff
@@ -210,6 +217,27 @@ private string CreateContent(FilePath file, string? mime, JObject pageModel)
         return ProcessHtml(_errors, file, content);
     }
 
+    private void ProcessConceptualHtml(string html, ConceptualModel conceptual)
+    {
+        var wordCount = 0L;
+
+        var result = HtmlUtility.TransformHtml(html, (ref HtmlReader reader, ref HtmlWriter writer, ref HtmlToken token) =>
+        {
+            HtmlUtility.AddLocaleIfMissingForAbsolutePath(ref token, _locale);
+
+            if (token.Type == HtmlTokenType.Text)
+            {
+                if (!_config.DryRun)
+                {
+                    wordCount += WordCount.CountWord(token.RawText.Span);
+                }
+            }
+        });
+
+        conceptual.Conceptual = LocalizationUtility.AddLeftToRightMarker(_cultureInfo, result);
```

Comment:
>AddLeftToRightMarker [](http://example.com/codeflow?start=52&length=20)

Not related to this PR: Is this where we should apply the `<ltr>` to `no-loc`? @AngryBerryMS